src/main/java/io/javalin/Context.kt

```diff
@@ -268,39 +527,89 @@ class Context(private val servletResponse: HttpServletResponse,
 
     // Translator methods
     // TODO: Consider moving rendering to JavalinServlet, where response is written
+    /**
+     * Render json object as the response result.
```

Comment:
Sets content-type to JSON and result to the JSON representation of a given object (using Jackson ObjectMapper)